### PR TITLE
Toggle between tables and charts for contribution data.

### PR DESCRIPTION
**Pull request by JonathanSR**
Adds functionality to toggle between tables and charts visualization for contributions data.
Pulled master into local, one test erroring out, local machine environment probable cause.

*Adds total contributions and industry breakdown charts.

### CODE REVIEW

File: components/bower_components/chartkick/.bower.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "chartkick",
```
Comment (TheKaterTot) [inline components/bower_components/chartkick/.bower.json]: I'm happy every time I see chartkick 👯

File: public_officials/static/public_officials/js/legislator_detail.js [hunk 2]
```diff
@@ -24,16 +25,47 @@ var getIndustryContributors = function(id){
...
+    var pacs = parseInt(industry['@attributes'].pacs)
+    indiv_total.push(indiv)
+    pac_total.push(pacs)
+  })
+  var indiv_sum = ["Individual Donations", (indiv_total.reduce((a, b) => a + b, 0))]
+  var pac_sum = ["Pac Donations", (pac_total.reduce((a, b) => a + b, 0))]
+  data.push(indiv_sum)
+  data.push(pac_sum)
+  new Chartkick.PieChart("total-donations", data, {is3D: true})
+}
```
Comment (TheKaterTot) [inline public_officials/static/public_officials/js/legislator_detail.js]: 🔥 🚒  you're on fire!

File: public_officials/static/public_officials/js/legislator_detail.js [hunk 3]
```diff
@@ -24,16 +25,47 @@ var getIndustryContributors = function(id){
...
+    var indiv = parseInt(industry['@attributes'].indivs)
+    var pacs = parseInt(industry['@attributes'].pacs)
+    indiv_total.push(indiv)
+    pac_total.push(pacs)
+  })
+  var indiv_sum = ["Individual Donations", (indiv_total.reduce((a, b) => a + b, 0))]
+  var pac_sum = ["Pac Donations", (pac_total.reduce((a, b) => a + b, 0))]
+  data.push(indiv_sum)
+  data.push(pac_sum)
+  new Chartkick.PieChart("total-donations", data, {is3D: true})
```
Comment (meyerhoferc) [inline public_officials/static/public_officials/js/legislator_detail.js]: 👍 is3D is the coolest option

File: public_officials/static/public_officials/js/legislator_detail.js [hunk 4]
```diff
@@ -24,16 +25,47 @@ var getIndustryContributors = function(id){
...
     var total = industry['@attributes'].total
     var industry = [name, total]
     data.push(industry)
-  })
+  });
   new Chartkick.PieChart("industry-donations", data)
-}
+};
+
+var industryBreakdownChart = function(data){
```
Comment (TheKaterTot) [inline public_officials/static/public_officials/js/legislator_detail.js]: now it's time for a breakdown 🎵

File: public_officials/static/public_officials/js/legislator_detail.js [hunk 5]
```diff
@@ -24,16 +25,47 @@ var getIndustryContributors = function(id){
...
+  var name = data.industry_name
+  var id   = data.industry_code
+  $("#industry-data-charts").append(`<h4>${name}</h4>`)
+  $("#industry-data-charts").append(`<div id=${id}></div>`)
+  var indiv = ["Individual Donations", (data.indivs || 0)]
+  var pac = ["PAC Donations", (data.pacs || 0)]
+  var chartData = []
+  chartData.push(indiv)
+  chartData.push(pac)
+  new Chartkick.PieChart(id, chartData)
```
Comment (meyerhoferc) [inline public_officials/static/public_officials/js/legislator_detail.js]: 💯

File: public_officials/static/public_officials/js/legislator_detail.js [hunk 6]
```diff
@@ -160,4 +200,7 @@ $(document).ready(function(){
   getSponsoredBills(legislator_pid);
   getVotingHistory(legislator_pid);
   $('.follow').on('click', followLegislator);
+  $('.btn-org').on('click', swapData('organization', 'data-charts'))
+  $('.btn-industry').on('click', swapData('industry-total', 'industry-data-chart'))
+
```
Comment (meyerhoferc) [inline public_officials/static/public_officials/js/legislator_detail.js]: awesome work @JonathanSR toggling is the best 🎉

File: public_officials/static/public_officials/js/legislator_detail.js [hunk 7]
```diff
@@ -146,10 +178,18 @@ var followLegislator = function() {
...
-var unfollowLegislator = function(legId, uId) {
-  $.post('/remove-follower', {lid: legId, uid: uId})
-  return $('.follow').text('Follow')
-}
+var swapData = function(div1, div2){
+  d1 = document.getElementById(div1)
+  d2 = document.getElementById(div2)
+  if(d2.style.display == "none")
+  {
+    d1.style.display = "none";
```
Comment (TheKaterTot) [inline public_officials/static/public_officials/js/legislator_detail.js]: 🎆

File: public_officials/static/public_officials/js/legislator_detail.js [hunk 8]
```diff
@@ -160,4 +200,7 @@ $(document).ready(function(){
   getSponsoredBills(legislator_pid);
   getVotingHistory(legislator_pid);
   $('.follow').on('click', followLegislator);
+  $('.btn-org').on('click', swapData('organization', 'data-charts'))
+  $('.btn-industry').on('click', swapData('industry-total', 'industry-data-chart'))
```
Comment (TheKaterTot) [inline public_officials/static/public_officials/js/legislator_detail.js]: Nice events handlers 👍

**Pull request merged**